### PR TITLE
Adds Feature Peer is Typing Notification

### LINKED ISSUE #35: Peer is typing notification
When a peer is typing, a notification should pop up to let the other person know.

**Relevant code pointers:**
Frempco: 
- Front end: Emit user typing socket to server from [SendMessages](https://github.com/mssiegel/frempco-client/blob/main/src/components/pages/StudentsPage/SendMessages.tsx)

4ScoreChat: 
- [Emit user is typing from front end](https://github.com/mssiegel/4Score-client/blob/master/src/components/chatbox/SendMessageForm.js)
- [Listen for user is typing on front end](https://github.com/mssiegel/4Score-client/blob/master/src/components/chatbox/Chatbox.js)
- [Server code - event is called 'typing'](https://github.com/mssiegel/4Score-server/blob/main/socketIOSetup/socketIOSetup.js) (on line 82)

Example from 4ScoreChat.
image

### PR DESCRIPTION
**Pull request by zachlankton**
Adds Feature: Peer is Typing Notification
Closes #35 

This PR is accompanied by a backend Server PR: https://github.com/mssiegel/frempco-server/pull/6

### CODE REVIEW

File: src/components/pages/StudentsPage/Conversation.tsx [hunk 1]
```diff
@@ -49,14 +62,52 @@ export default function Conversation({
...
         else if (person === 'you') fontCSS = conversationCSS.you;
-
         return (
           <Typography key={i}>
             <span css={fontCSS}>{filterWords(character)}: </span>
             <span>{filterWords(message)}</span>
           </Typography>
         );
       })}
+      {peerIsTyping && (
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/Conversation.tsx:72]: The typing message should appear right above sendMessageForm in very bottom of chat.

File: src/components/pages/StudentsPage/Conversation.tsx [hunk 2]
```diff
@@ -49,14 +62,52 @@ export default function Conversation({
...
+            <TypingDots />
+          </span>
+        </Typography>
+      )}
+      <span ref={lastMessage} />
     </Box>
   );
 }
+
+const TypingDots = (props) => {
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/Conversation.tsx:85]: Instead of typing dots, lets go for static ...

**Review by mssiegel — CHANGES_REQUESTED** [review]
After addressing above comments, go ahead and merge! This is amazing!

**Comment by zachlankton** [comment]
Included in this latest update was a couple minor CSS changes to also make sure the chat messages would break on long unbroken strings to prevent overflow in the X-axis

**Pull request merged**